### PR TITLE
chore: added Milan in OpenSource Projects

### LINKED ISSUE #545: [Add] Milan
### Site URL

https://github.com/IAmTamal/Milan

### Description of the above provided site

Milan is a hub to connect users, charities, and NGOs thus connecting help and need for a good cause.

### Category

Frontend

### Sub Category

Cat - OpenSource  ,   SubCat - Projects 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #545 

## Changes proposed

Milan is a hub to connect users, charities, and NGOs thus connecting help and need for a good cause.

## Screenshots

Screenshot (802)

1) Updated description boxicons
Screenshot (803)

2) Updated description of Milan
Screenshot (805)

## Note to reviewers

Any kind of feedback would be appreciated.

### CODE REVIEW

**Comment by rupali-codes** [comment]
Hey Parma, thanks a lot, could you please update the title of this PR to something meaningful like "chore: added Milan in OpenSource Projects"

**Comment by parma1111** [comment]
@rupali-codes @CBID2 @ujjawaltyagii

File: database/opensource/projects.json [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+  {
+    "name": "LinkFree",
+    "description": "An Open Source alternative to LinkTree to showcase your projects, milestones, events, and content.",
+    "url": "https://github.com/EddieHubCommunity/LinkFree",
+    "category": "opensource",
+    "subcategory": "projects"
+  },
+  {
+    "name": "Milan",
+    "description": "Milan is a hub to connect users, charities, and NGOs thus connecting help and need for a good cause.",
```
Comment (CBID2) [inline database/opensource/projects.json:11]:
```suggestion
    "description": "Milan is the place to connect users, charities, and NGOs thus connecting those who need help and need for a good cause.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @parma111! :) I just made some suggestions on how to improve your description. Other than that, good job!

**Comment by parma1111** [comment]
> Hey @parma111! :) I just made some suggestions on how to improve your description. Other than that, good job!

>Hey @CBID2 !:) I have changed the description

File: database/frontend/icons.json [hunk 2]
```diff
@@ -104,6 +104,13 @@
     "category": "frontend",
     "subcategory": "icons"
   },
+  {
+    "name": "Boxicons",
+    "description": "Boxicons offer 2,000+ elegant vector icons for websites, apps, and digital projects. Clean lines and easy customization.",
```
Comment (CBID2) [inline database/frontend/icons.json:109]:
```suggestion
    "description": "Boxicons offer 2,000+ clean, elegant, and easily-customizable vector icons for websites, apps, and digital projects.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @parma1111! :) I just made another suggestion to your PR! :) Keep it up! :)

**Comment by parma1111** [comment]
> Hi @parma1111! :) I just made another suggestion to your PR! :) Keep it up! :)

> Hi @CBID2 ! :) Now review the PR.

File: database/frontend/icons.json (hunk 2 above)
Comment (parma1111) [reply database/frontend/icons.json]: changed the description

**Review by CBID2 — APPROVED** [review]
Looks great @parma1111! Now, LGTM!!:)

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm :)

**Pull request merged**